src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public Colour getRandomColor() {
```

Comment:
The method name 'getRandomColor' suggests that the return type should be 'Color', but it is currently 'Colour'. Ensure that the return type matches the class name.

Reply:
Fixed